src/main.rs

```diff
@@ -869,4 +914,213 @@ mod tests {
             assert_eq!(response.status(), StatusCode::BAD_REQUEST);
         }
     }
+
+    mod config {
```

Comment:
It's inside of `tests`, but separate since the other tests are in `tests::attest`, this is `tests::config`

Reply:
Avoid nesting, just put them in `tests`. If it's too big, create a directory `/tests` with `/tests/config.rs` and `/tests/FEATURE.rs`

Reply:
Deferring unit test cleanup as https://github.com/profianinc/steward/issues/134 since this PR is already quite large and the existing unit tests were already quite large and cumbersome.